src/applications/vaos/services/mocks/index.js

```diff
@@ -25,15 +25,18 @@ const requestsV2 = require('./v2/requests.json');
 // const requestsV2 = require('./v2/requests_null_states.json.json');
 
 // Uncomment to produce backend service errors
-// const meta = require('./v2/meta_failures.json');
+const meta = require('./v2/meta_failures.json');
```

Comment:
Should we revert this after testing? It doesn't sound like we need this permanently right? Same with a few other spots in this file.

Reply:
Thanks for the feedback @JunTaoLuo 
Commented the lines to revert back before testing.